app/policies/consumer_role_policy.rb

```diff
@@ -50,6 +50,12 @@ def edit?
     return false
   end
 
+  # Checking presence of hbx_staff_role and if identity_validation is valid. If either are true,
+  # then the user has access to continue past RIDP.
+  def ridp_accessible?
```

Comment:
Below are the type of users who needs to have access to this endpoint.
- Admin Access - `hbx_staff_role.permission.modify_family`
- Broker Access - Brokers are assigned to families to access the Consumer Accounts. We need to check if the Broker's Agency Staff Role is active and is associated with the family.
- Account Access by the Family itself.

Reply:
https://github.com/ideacrew/enroll/pull/3573/files#diff-437aeaa6d71572bcc20d9566d96f9918b4a1ef2640a5d6dd60989bc138703d8dR53-R61